toshi_hazard_post/hazard_aggregation/locations.py

```diff
@@ -0,0 +1,117 @@
+from typing import Dict, List, Tuple
+
+from nzshm_common.grids.region_grid import load_grid
+from nzshm_common.location.code_location import CodedLocation
+from nzshm_common.location.location import LOCATIONS_BY_ID
+
+
+def get_locations(config):
+
+    if config.locations == "NZ_34":
+        locations = [(loc['latitude'], loc['longitude']) for loc in LOCATIONS_BY_ID.values()]
+    else:
+        locations = (
+            load_grid(config.locations)
+            if not config.location_limit
+            else load_grid(config.locations)[: config.location_limit]
+        )
+
+    return locations
+
+
+def locations_by_degree(
```

Comment:
@chrisdicaprio I was thinking these location helpper functions should be in nzhsm-common rather than THS or THP?